actions/transfer-case.vbs

```diff
@@ -138,466 +178,296 @@ EMReadScreen mail_addr_line_four, 43, 12, 27
-	IF crf_sent_checkbox = CHECKED THEN
-		crf_sent_date = date
-		call write_bullet_and_variable_in_case_note("Change Report Sent", crf_sent_date)
-		call write_bullet_and_variable_in_case_note("Case File Sent:", crf_sent_date)
-	END IF
-	IF excluded_time = "Yes" THEN
-		excluded_time = excluded_time & ", Begins " & excluded_date
-		call write_bullet_and_variable_in_case_note("Excluded Time" , excluded_time)
-		excluded_time = "Yes"
-	ELSEIF excluded_time = "No" THEN
-		call write_bullet_and_variable_in_case_note("Excluded Time", excluded_time)
-	ELSEIF excluded_time = "" THEN
-		call write_variable_in_case_note("* Excluded Time: N/A")
-	END IF
-	IF hc_status_check = "ACTV" THEN
-		CALL write_bullet_and_variable_in_case_note("HC County of Financial Responsibility", hc_cfr)
-		IF hc_cfr_no_change_checkbox = 0 THEN
-			CALL write_bullet_and_variable_in_case_note("HC CFR Change Date", (cfr_month & "/" & cfr_year))
-		ELSE
-			CALL write_bullet_and_variable_in_case_note("HC CFR", "Not changing")
-		END IF
-	END IF
-	IF cash1_status_check = "ACTV" OR cash2_status_check = "ACTV" THEN
-		CALL write_bullet_and_variable_in_case_note("CASH County of Financial Responsibility", county_financial_responsibilty)
-		IF cash_cfr_no_change_checkbox = 0 THEN
-			CALL write_bullet_and_variable_in_case_note("CASH CFR Change Date", (cfr_month & "/" & cfr_year))
-		ELSE
-			CALL write_bullet_and_variable_in_case_note("CASH CFR", "Not changing")
-		END IF
-	END IF
-	IF closure_date_checkbox = CHECKED THEN call write_variable_in_case_note("* Client has until " & closure_date & " to provide required proofs or the case will close.")
-	If transfer_form_checkbox = CHECKED THEN call write_variable_in_case_note("* DHS 3195 Inter Agency Case Transfer Form completed and sent.")
-	IF SPEC_MEMO_checkbox = CHECKED THEN call write_variable_in_case_note("* SPEC/MEMO sent to client with new worker information.")
-	IF forms_to_arep = "Y" THEN call write_variable_in_case_note("* Copy of SPEC/MEMO sent to AREP.")
-	IF forms_to_swkr = "Y" THEN call write_variable_in_case_note("* Copy of SPEC/MEMO sent to social worker.")
+    Call write_bullet_and_variable_in_CASE_NOTE("Actions to be taken", action_to_be_taken)
+    Call write_bullet_and_variable_in_CASE_NOTE("Requested verifications", requested_verifs)
+    Call write_bullet_and_variable_in_CASE_NOTE("Expected changes in household's circumstances:", expected_changes)
+    Call write_bullet_and_variable_in_CASE_NOTE("Other notes", other_notes)
+    IF mets_status_dropdown = "Active" THEN Call write_bullet_and_variable_in_CASE_NOTE("* Client is active on HC through METS case number:", METS_case_number)
+    call write_bullet_and_variable_in_case_note("Client move date", client_move_date)
+    call write_bullet_and_variable_in_case_note("Change report sent", date) 'defaulting information '
+    call write_bullet_and_variable_in_case_note("Case file sent:", date) 'defaulting information '
+    IF excluded_time = "Yes" THEN
```

Comment:
There is still some variable management needed here. We had talked about changing it to:
```
IF excluded_time_dropdown = "Yes" THEN
call write_bullet_and_variable_in_case_note("Excluded Time" , "Yes, Begins " & excluded_date)
ELSEIF excluded_time_dropdown = "No" THEN
call write_bullet_and_variable_in_case_note("Excluded Time", excluded_time_dropdown)
END IF
```